### PR TITLE
Reformat code with modified imglib2 formatter template

### PR DESCRIPTION
**Pull request by stefanhahmann**
* the intention of this pull request is that whole code basis is formatted with the formatting template to be used in the future allowing afterwards auto-format to be switched on in IDEs without this leading to troubles in future pull-requests
* new formatter template will be here: https://github.com/scijava/scijava-coding-style/pull/6
  * formatter template will ensure that long comments are preserved in one line
* adds some information for users and contributors of Mastodon to READEM.md and new file CONTRIBUTING.md

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+We use the Eclipse code formatter. You can find the configuration
+file [here](https://github.com/scijava/scijava-coding-style/blob/master/src/main/resources/eclipse-formatter-settings/mastodon-coding-style.xml)
+. Please use this formatter for your pull requests.
+
+Users of IntelliJ IDEA can use
+the [Eclipse Code Formatter Plugin](https://plugins.jetbrains.com/plugin/6546-eclipse-code-formatter) to automatically
+format your code.
+
+We recommend to use Auto-Format on Save in your IDE, preferably only on changed lines.
```
Comment (tinevez) [inline CONTRIBUTING.md:21]: This is really good! And useful!
What do you think of actually putting this in the org page, and say that it applies to *all* mastodon repos?

File: README.md [hunk 2]
```diff
@@ -167,3 +167,17 @@ If TrackScheme is fully zoomed, a tap of `Z` unzoom fully.
...
+
+# Community
+
+* If you have questions regarding usage of Mastodon, we invite you to use
+  the [forum on imagesc](https://forum.image.sc/). You may search for Mastodon related topics using the
+  hashtag `#mastodon` or create a new topic.
+
+## Contribute
+
+Read through [how to contribute](CONTRIBUTING.md) for information on how to contribute to this project.
```
Comment (tinevez) [inline README.md:183]: The same. Maybe move it to the umbrella org page.

**Review by tinevez — COMMENTED** [review]
Ok I give up on vetting each individual change one by one in this review. 
The only changes I would suggest is to put the contributing and readme edits into the main org, so that this applies to ALL mastodon project under this umbrella.

File: CONTRIBUTING.md (hunk 1 above)
Comment (stefanhahmann) [reply CONTRIBUTING.md]: Good suggestion.
I removed the code from here and made a pull-request to the org page.
@tinevez Please have a look at: https://github.com/mastodon-sc/.github/pull/1

File: README.md (hunk 2 above)
Comment (stefanhahmann) [reply README.md]: Good suggestion.
I removed the code from here and made a pull-request to the org page.
@tinevez Please have a look at: https://github.com/mastodon-sc/.github/pull/1

**Pull request merged**